web/src/components/ImageSurveyCell/ImageSurveyCell.tsx

```diff
@@ -194,29 +214,43 @@ const ImageSurveyComponent = ({
             ]}
             direction="column"
             value={imageSurvey?.privateRank.toString() || ''}
-            validation={{ required: true }}
+            validation={{ required: 'Image privacy question is required' }}
+            errorClassName="error"
           />
           <OpenEndedInputTagField
-            tags={tagsPublic}
             onTagsChange={handleTagsChangePublic}
             placeholder="Answer here"
-            value={imageSurvey?.privateElem || ''}
+            value={{
+              tags: imageSurvey?.publicElem?.split(',') || ([] as string[]),
```

Comment:
The tags are define as: `["hello", "text"]` and  they are store as: `"hello,text"`
So it is split on the `,`.

Reply:
Ah because of the conversion form an array to a string with with `.toString()` ? Try inputting "hello world,hello". It will produce 2 tags on input: `["hello", "world,hello"]` but when you will retrieve it, and parse it, it will give 3: `["hello", "world", "hello"]` ...

Reply:
I change the data as you suggest in the previous comment, so the split is in the " " and it resolves this problem.